mobile/src/bundles/common/components/selector/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { StyleSheet } from 'react-native';
+
+import { Color } from '../../enums/enums';
+
+const styles = StyleSheet.create({
+    dropdownButton: {
+        borderWidth: 1,
+        borderColor: Color.INPUT,
+    },
+    dropdown: {
+        position: 'absolute',
+        width: '100%',
+        backgroundColor: '#FFF',
+        zIndex: 999,
```

Comment:
Let's set the zIndex value to 1 since this component doesn't need to be at the most top level.  (We'll have Overlay  component )